doc/tutorials/getting_started/getting_started.rst

```diff
@@ -85,6 +85,15 @@ Optional: add the previous command to your ``.bashrc``: ::
    not required and often skipped by advanced users who use more than one
    Colcon workspace at a time, but we recommend it for simplicity.
 
+Switch to Cyclone DDS
+^^^^^^^^^^^^^^^^^^^^^
+
+As of Sep 26, 2022, the default ROS2 DDS has an issue which is fixed by switching to Cyclone:
```

Comment:
Pedantic, but the issue isn't really *fixed*, insomuch as it's worked around by avoiding the RMW with the issue completely.